paper-tab.html

```diff
@@ -43,11 +44,15 @@
       position: relative;
       padding: 0 12px;
       overflow: hidden;
-      cursor: default;
+      cursor: pointer;
 
       @apply(--paper-tab);
     }
 
+    :host(:focus) {
```

Comment:
I'm going with what @blasten did for paper-menu, using bolded text to indicate an additional focus state (where the states are not-focused + not-selected, focused + not-selected and focused + selected). The outline was not showing up very well in the first place, so I am hopeful that this is an improvement.